src/ios/SOSPicker.m

```diff
@@ -74,25 +74,24 @@ - (void) getPictures:(CDVInvokedUrlCommand *)command {
 
 - (void)launchGMImagePicker:(bool)allow_video title:(NSString *)title message:(NSString *)message disable_popover:(BOOL)disable_popover maximumImagesCount:(NSInteger)maximumImagesCount
 {
-    GMImagePickerController *picker = [[GMImagePickerController alloc] init:allow_video];
-    picker.delegate = self;
-    picker.maximumImagesCount = maximumImagesCount;
-    picker.title = title;
-    picker.customNavigationBarPrompt = message;
-    picker.colsInPortrait = 4;
-    picker.colsInLandscape = 6;
-    picker.minimumInteritemSpacing = 2.0;
-
+    self.picker = [[GMImagePickerController alloc] init:allow_video];
```

Comment:
This can be non-global if we can access the picker from the presentationController (Should test first).